Mage.Tests/src/test/java/org/mage/test/cards/triggers/damage/ObNixilisCaptiveKingpinTest.java

```diff
@@ -0,0 +1,174 @@
+package org.mage.test.cards.triggers.damage;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+public class ObNixilisCaptiveKingpinTest extends CardTestCommander4Players {
+
+    @Test
+    public void damageController1Point() {
+        addCard(Zone.BATTLEFIELD, playerA, "Ob Nixilis, Captive Kingpin", 1);
+        addCard(Zone.BATTLEFIELD, playerD, "Memnite");
+
+        attack(2, playerD, "Memnite", playerA);
+
+        setStopAt(2, PhaseStep.END_TURN);
```

Comment:
Miss setStrictChooseMode before setstop/execute. It allows to check commands order and missing commands. Must be used all the time.